projects/jazz-lib/README.md

```diff
@@ -0,0 +1,24 @@
+# JazzLib
```

Comment:
You probably want a more descriptive name here: "Princeton Design System Angular Library"